pages/view/_iscnId.vue

```diff
@@ -304,6 +304,7 @@
             class="mb-[12px]"
           >
             <Link
+              v-if="urlPattern.test(metadata.usageInfo)"
```

Comment:
make this as a computed prop, then you don't have to inject `URL_REGEX` into data